src/pages/MoviesList/MovieApp.jsx

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable prefer-const */
+import React, { useEffect, useState } from "react";
+import MovieList from "components/MovieList/MovieList";
+import Footer from "components/Footer/Footer";
```

Comment:
Consider organizing imports alphabetically, this makes them easier to manage and find. Also, we normally use single quotes for strings instead of double quotes. 

To help organize imports alphabetically, and using single quotes, you can configure eslint/prettier to help you with that automatically on save. I advise reading the following article on how to configure those:
https://medium.com/dooboolab/using-eslint-prettier-and-sort-imports-vscode-extensions-for-formatting-open-source-project-16edf317129d